app/src/main/java/com/android/androidfundamentalsgroup1/MainActivity.java

```diff
@@ -3,19 +3,36 @@
 import androidx.appcompat.app.AppCompatActivity;
 
 import android.os.Bundle;
+import android.util.Patterns;
 import android.view.View;
 import android.widget.Button;
 import android.widget.CheckBox;
 import android.widget.CompoundButton;
 import android.widget.EditText;
 import android.widget.SeekBar;
 import android.widget.TextView;
+import android.widget.Toast;
 
 public class MainActivity extends AppCompatActivity {
 
+    private EditText emailInput;
+    private EditText phoneInput;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
+        setContentView(R.layout.login_view);
+    }
+
+    public void buttonSubmitOnClick(View view){
+        emailInput = findViewById(R.id.emailInput);
```

Comment:
The code challenge mentioned also to display the data from edit text and checkbox in a textview